docs/en/integrations/dbt/dbt-incremental-model.md

```diff
@@ -244,4 +244,107 @@ Adjust the above query to the period of execution. We leave result inspection to
     from actor_summary__dbt_tmp;
     ```
 
-This strategy may encounter challenges on very large models. For further details see [Limitations](./dbt-limitations).
\ No newline at end of file
+This strategy may encounter challenges on very large models. For further details see [Limitations](./dbt-limitations).
+
+
+
+## Inserts-only mode
+
+To overcome the limitations of large datasets in incremental models, the plugin offers a configuration params ‘inserts-only’. When set, updated rows are inserted directly to the target table (a.k.a imdb_dbt.actor_summary) and no temporary table is being created.
+Note: Insert-only mode requires your data to be immutable. If you want an incremental table model that supports altered rows don’t use this mode!
+
+To illustrate this mode, we will add another new actor and re-execute dbt run with inserts_only=True
+
+1. Configure inserts_only mode in actor_summary.sql:
+
+   ```sql
+   {{ config(order_by='(updated_at, id, name)', engine='MergeTree()', materialized='incremental', unique_key='id', inserts_only=True) }}
+   ```
+   
+2. Let’s add another famous actor - Danny DeBito
+
+   ```sql
+   INSERT INTO imdb.actors VALUES (845467, 'Danny', 'DeBito', 'M');
+   ```
+
+3. Let's star Danny in 920 random movies.
+
+   ```sql
+   INSERT INTO imdb.roles
+   SELECT now() as created_at, 845467 as actor_id, id as movie_id, 'Himself' as role
+   FROM imdb.movies
+   LIMIT 920 OFFSET 10000;
+   ```
+
+4. Execute a dbt run and confirm that Danny was added to the actor-summary table
+
+   ```bash
```

Comment:
```suggestion
   ```response
```

Reply:
This is a bash command, why changing this to response? 
We have 2 similar blocks above with bash annotators

Reply:
This block can't be copied and pasted to run the command because it includes the prompt and the output, by changing it from bash to response it removes the "copy" button from the page.